include/fs/fs.h

```diff
@@ -170,20 +185,29 @@ struct fs_file_system_t {
 #define FS_SEEK_END	2	/* Seek from end of file.  */
 #endif
 
-
 /**
  * @brief File open
  *
- * Opens an existing file or create a new one and associates
- * a stream with it.
+ * Opens or crates, if does not exist file, depending on mode provided
+ * and associates a stream with it.
  *
  * @param zfp Pointer to file object
  * @param file_name The name of file to open
+ * @param flags The mode flags
+ *
+ * @p flags can be empty, or combination of one or more of following flags:
+ *   FS_O_READ    open for read
+ *   FS_O_WRITE   open for write
+ *   FS_O_RDWR    open for read/write (<tt>FS_O_READ | FS_O_WRITE</tt>)
+ *   FS_O_CREATE  create file if it does not exist
+ *   FS_O_APPEND  open file at the end
```

Comment:
This is incorrect.  POSIX semantics is "move to end before each write".  The file position is not moved to the end on open.

Please also fix in commit message.

Reply:
Done